bitcoin-extensions/bitcoin.html

```diff
@@ -0,0 +1,33 @@
+<html>
+	<head>
+		<title>Bitcoin Price Index Watcher</title>
+		<link href="styles.css" rel="stylesheet" type="text/css">
+	</head>
+	<body>
+		<img id="logo" src="bitcoin.png" />
+		<div id="data"></div>
+		<script type="text/javascript">
+			var xmlhttp = new XMLHttpRequest();
+			var url = "https://api.coindesk.com/v1/bpi/currentprice.json";
+
+			xmlhttp.onreadystatechange = function() {
+				if (this.readyState == 4 && this.status == 200) {
+					var json = JSON.parse(this.responseText);
+					parseJson(json);
+				}
+			};
+
+			xmlhttp.open("GET", url, true);
+			xmlhttp.send();
+
+			function parseJson(json) {
+				var time = "<b>Last Updated: " + json["time"]["updated"] + "</b>"
```

Comment:
Using let and const instead of var here would easily communicate to another developer if those values are meant to change or stay constant.